rotkehlchen/data/importer.py

```diff
@@ -240,6 +240,7 @@ def _consume_cryptocom_entry(self, csv_row: Dict[str, Any]) -> None:
             'referral_card_cashback',
             'card_cashback_reverted',
             'reimbursement',
+            'viban_purchase',
```

Comment:
is this a typo? What is viban? I see it below too. So probably not a typo. What is it?

Reply:
I saw this kind of transaction at #3056 By a search on google seems to be purchases using a fiat wallet